README.md

```diff
@@ -1,38 +1,63 @@
+<p align="center">
+    <a href="http://kitura.io/">
+        <img src="https://raw.githubusercontent.com/IBM-Swift/Kitura/master/Sources/Kitura/resources/kitura-bird.svg?sanitize=true" height="100" alt="Kitura">
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://ibm-swift.github.io/FileKit/index.html">
+    <img src="https://img.shields.io/badge/apidoc-FileKit-1FBCE4.svg?style=flat" alt="APIDoc">
+    </a>
+    <a href="https://travis-ci.org/IBM-Swift/FileKit">
+    <img src="https://travis-ci.org/IBM-Swift/FileKit.svg?branch=master" alt="Build Status - Master">
+    </a>
+    <img src="https://img.shields.io/badge/os-macOS-green.svg?style=flat" alt="macOS">
+    <img src="https://img.shields.io/badge/os-linux-green.svg?style=flat" alt="Linux">
+    <img src="https://img.shields.io/badge/license-Apache2-blue.svg?style=flat" alt="Apache 2">
+    <a href="http://swift-at-ibm-slack.mybluemix.net/">
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg" alt="Slack Status">
+    </a>
+</p>
+
 # FileKit
 
-> Utility for resolving common paths
+Resolves commonly used paths, including the project, executable and working directories.
+
+## Version
+The latest release of `FileKit` (0.x.x) runs on Swift 4.0 and newer, on both macOS and Ubuntu Linux.
+
+## Usage
 
-[![Build Status - Master](https://travis-ci.org/IBM-Swift/FileKit.svg?branch=master)](https://travis-ci.org/IBM-Swift/FileKit)
-![macOS](https://img.shields.io/badge/os-macOS-green.svg?style=flat)
-![Linux](https://img.shields.io/badge/os-linux-green.svg?style=flat)
-![Apache 2](https://img.shields.io/badge/license-Apache2-blue.svg?style=flat)
+### Add dependencies
 
-## Version Info
-The latest release of `FileKit` (v0.x.x) runs on Swift 4.0 and newer, on both macOS and Ubuntu Linux
+Add `FileKit` to the dependencies within your application's `Package.swift` file. Substitute `"x.x.x"` with the latest `FileKit` [release](https://github.com/IBM-Swift/FileKit/releases).
 
-## Installation
+```swift
+.package(url: "https://github.com/IBM-Swift/FileKit.git", from: "x.x.x")
+```
+Add `FileKit` to your target's dependencies:
 
-### Via Swift Package Manager
+```Swift
+.target(name: "example", dependencies: ["FileKit"]),
+```
 
-Add `FileKit` to your `Package.swift`:
+#### Import package
 
 ```swift
+import FileKit
+```
 
-let package = Package(
-  name: "<package-name>",
-  dependencies: [
-    // Swift 4
-    .package(url: "https://github.com/IBM-Swift/FileKit.git", .upToNextMajor(from: "0.0.0"))
-  ]
-)
+You will also need to import the `Foundation` package if you're handling URLs:
 
+```swift
+import Foundation
 ```
 
 ## Supported Paths
 
 #### Path to Executable Folder
 
-Points to the project executable folder
+Points to the project executable folder.
```

Comment:
This is a bit vague - could we give a more specific description?  For example, is this the project directory containing the `.build`, or is it literally the location of the executable (eg. `/MyProj/.build/debug/MyProj`)?

Reply:
I've clarified the wording, added examples and the README and apidoc are now in sync.